bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java

```diff
@@ -137,6 +138,15 @@ private Cluster getCluster(BigtableClusterAdminClient client, String clusterName
     GetClusterRequest request = GetClusterRequest.newBuilder().setName(clusterName).build();
     try {
       return client.getCluster(request);
+    } catch (UncheckedExecutionException e) {
+      if (e.getCause() != null && e.getCause() instanceof StatusRuntimeException) {
```

Comment:
I think u can do without "e.getCause() != null". If the cause is null, instanceof will be false.